android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsBannerAdViewManager.java

```diff
@@ -135,11 +135,21 @@ public void setSizes(ReactNativeAdView reactViewGroup, ReadableArray value) {
       payload.putDouble("height", adSize.getHeight());
       sendEvent(reactViewGroup, EVENT_SIZE_CHANGE, payload);
     }
-
+    reactViewGroup.setSizesArray(value);
     reactViewGroup.setSizes(sizeList);
     reactViewGroup.setPropsChanged(true);
   }
 
+  @ReactProp(name = "maxAdHeight")
+  public void setMaxAdHeight(ReactNativeAdView reactViewGroup, float value) {
+    reactViewGroup.setMaxAdHeight(value);
+    ReadableArray adSizes = reactViewGroup.getSizesArray();
+    if (adSizes != null) {
+      this.setSizes(reactViewGroup, adSizes);
+    }
```

Comment:
From my testing in the example app, this condition is never true. So we might not need the extra sizesArray.